packages/-ember-data/tests/unit/many-array-test.js

```diff
@@ -105,6 +105,7 @@ module('unit/many_array - DS.ManyArray', function (hooks) {
     let willChangeRemoveAmt;
     let willChangeAddAmt;
 
+    DS.ManyArray.proto().__hasArrayObservers = true;
```

Comment:
Would be nice to have a comment here so we don't have to search later why we are doing this